src/main/resources/org/sonatype/nexus/ci/iq/PolicyEvaluationHealthAction/summary.groovy

```diff
@@ -57,6 +61,7 @@ t.summary(icon: '/plugin/nexus-jenkins-plugin/images/48x48/nexus-iq.png') {
     span(class: 'iq-chiclet moderate', action.moderateComponentCount)
   }
   if (!action.criticalComponentCount && !action.severeComponentCount && !action.moderateComponentCount) {
-    span(Messages.IqPolicyEvaluation_NoViolations())
+    span(class: 'iq-chiclet-message', Messages.IqPolicyEvaluation_NoViolations())
```

Comment:
Why was this class introduced?  Does it decrease or increase the default margin width?

Reply:
> Why was this class introduced? Does it decrease or increase the default margin width?

The short version is that we need some way to have the same right margin on the "No Violations" text as we would have if a chiclet were present instead. If we don't do that, then the new grandfathering message gets scrunched in with the "No Violations" text and it looks rather unsightly.

The most direct way I could think of doing that without refactoring more of the CSS was to introduce a new class for the "message"-like entries displayed along with the chiclets, as this one change seems to resolve the problem without necessitating any further changes. It has a `margin-right` of 5 pixels just like the normal `iq-chiclet` class, but without all the additional properties that are otherwise included with the latter class.

Compare:

https://github.com/sonatype/jenkins-nexus-platform-plugin/blob/83d16df8a4bc5f36953aaff4e576dc5f1d2367f8/src/main/resources/org/sonatype/nexus/ci/iq/PolicyEvaluationHealthAction/summary.groovy#L32

https://github.com/sonatype/jenkins-nexus-platform-plugin/blob/83d16df8a4bc5f36953aaff4e576dc5f1d2367f8/src/main/resources/org/sonatype/nexus/ci/iq/PolicyEvaluationHealthAction/summary.groovy#L48

Reply:
> Thanks for the explanation. Would a margin-left on Messages.IqPolicyEvaluation_NumberGrandfathered do the same?

It would. That's originally what I had, but it felt a little more ad-hoc than this approach, particularly since if anyone ever had to add anything else to this line they'd end up running into a very similar problem (much like we are in this particular case). I'd rather have a class that enforces the same margin-right as the chiclets so you can add things interchangeably and get the same spacing between the elements.

That said, if you're uncomfortable with this approach, I can go back and change it. Perhaps if I understood your concern regarding the margin-right things would be more clear to me?

Reply:
>It would. That's originally what I had, but it felt a little more ad-hoc than this approach, particularly since if anyone ever had to add anything else to this line they'd end up running into a very similar problem (much like we are in this particular case). I'd rather have a class that enforces the same margin-right as the chiclets so you can add things interchangeably and get the same spacing between the elements.

I think what you've done here is good.  Was just thinking that margin-left would reduce the changes, but now I understand the reasoning behind it.  Thanks.